src/main/java/com/nexmo/quickstart/sms/ReceiveDLR.java

```diff
@@ -0,0 +1,27 @@
+package com.nexmo.quickstart.sms;
+
+import static spark.Spark.*;
+
+import static com.nexmo.quickstart.Util.configureLogging;
+
+public class ReceiveDLR {
+
+    public static void main(String[] args) throws Exception {
+        configureLogging();
+
+        port(8080);
+
+        get("/receipt", (req, res) -> {
```

Comment:
We might want to mention in the README that this repo is Java 7+. Otherwise, looks good to me.